docs/griptape-cloud/knowledge-bases/accessing-data.md

```diff
@@ -0,0 +1,33 @@
+# Accessing Data in a Knowledge Base
+
+You can `Search` or `Query` the Knowledge Base for information contained in your Data Sources. `Search` will return a natural language response while `Query` will return the individual entries. Use whichever one best fits your use case.
+
+## From the Cloud Console
+
+You can try out both actions from the cloud console on the `Test` tab of your Knowledge Base.
```

Comment:
Can we explain what "try out" means here? We're letting people explore their data, I believe.

Reply:
Does the top description not cover that? or are we looking for something more descriptive?

> You can `Search` or `Query` the Knowledge Base for information contained in your Data Sources. `Search` will return a natural language response while `Query` will return the individual entries. Use whichever one best fits your use case.

Reply:
I'd just re-iterate and get a little pedantic. "You can explore your data with natural language on the Test tab. Compare and contrast the results of Search vs. Query to understand which is correct for your application."